src/constructs/core/stack.ts

```diff
@@ -127,7 +137,13 @@ export class GuStack extends Stack implements StackStageIdentity {
 
   // eslint-disable-next-line custom-rules/valid-constructors -- GuStack is the exception as it must take an App
   constructor(scope: App, id: string, props: GuStackProps) {
-    const { cloudFormationStackName = process.env.GU_CFN_STACK_NAME, stack, stage, withoutTags } = props;
+    const {
+      cloudFormationStackName = process.env.GU_CFN_STACK_NAME,
+      stack,
+      stage,
+      withoutTags,
+      withoutBackup = false,
```

Comment:
While I appreciate that `withoutBackup` is consistent with the existing `withoutTags` property, I found the double negative `withoutBackup = false` hard to parse 😅

Reply:
Totally fair!

Can think of two alternatives:
1. Change the type of `withoutBackup` to `withoutBackup?: true` - when undefined, backups are enabled, when `true` backups are disabled
2. Change the property name to `withBackup`

Though it breaks consistency with the other properties, I think the second option is better. WDYT?

Reply:
Yeah, I agree, I think using `withBackup` would make it easier to read!